uninstall.sh

```diff
@@ -1,24 +1,44 @@
 #!/usr/bin/env bash
 
+echo 'Hi there!'
+echo 'This script 1. uninstalls a hyprland end-4 dot install, installed using the script provided in the repo.'
+echo '            2. is not tested, use at your own risk.'
+echo '            3. will show all commands that it runs.'
+echo '            4. should be run from its folder.'
+echo 'Ctrl+C to exit. Enter to continue.'
+read
+
+##############################################################################################################################
+
 # Undo Step 3: Removing copied config and local folders
 echo 'Removing copied config and local folders...'
+echo 'rm -rf "$HOME/.config/ags fish frontconfig foot fuzzel gtklock hypr mpv swaylock wlogout starship.toml" '

```

Comment:
this syntax won't work
remove each folder in one command

Reply:
actually, no need to remove each in a command
just have multiple targets in one line but not this way
something like `rm -rf "$HOME/.config/ags" "$HOME/.config/fish"`

Reply:
Fixed with commit [91dc06d](https://github.com/end-4/dots-hyprland/pull/148/commits/91dc06dd1b7272b0516c86b881ca649a777557d5)